src/components/Utils/PrivateRouteShelter.js

```diff
@@ -5,5 +5,5 @@ import AuthService from '../../services/authService';
 export default function PrivateRouteShelter({ children }) {
   return AuthService.isLoggedIn() && AuthService.isShelterAdmin()
```

Comment:
What is the difference between AuthService.isLoggedIn() and AuthService.isShelterAdmin()? If `PrivateRouteShelter.js` is called, isn't it always ShelterAdmin? If we ended up combining `PrivateRoute.js` and `PrivateRouteShelter.js`, I think we can distinguish those by passing props...but I might be wrong. Let me know if I am missing something!

Reply:
isLoggedIn() makes sure the user is logged in, isShelterAdmin() makes sure the logged in user is an admin. We could probably just have isShelterAdmin() here instead of both, if that is what you were asking about

Reply:
Yeah I was thinking that maybe we can just have one. The way you implemented works fine and I think it's less confusing. Thanks for the explanation!